components/size_aware_image.jsx

```diff
@@ -108,22 +130,76 @@ export default class SizeAwareImage extends React.PureComponent {
         return null;
     }
 
-    renderImageOrPlaceholder = () => {
+    renderImageWithContainerIfNeeded = () => {
         const {
-            dimensions,
             fileInfo,
             src,
             ...props
         } = this.props;
 
-        let placeHolder;
-        let imageStyleChangesOnLoad = {};
+        Reflect.deleteProperty(props, 'showLoader');
+        Reflect.deleteProperty(props, 'onImageLoaded');
+        Reflect.deleteProperty(props, 'onImageLoadFail');
+        Reflect.deleteProperty(props, 'dimensions');
+        Reflect.deleteProperty(props, 'handleSmallImageContainer');
+
         let ariaLabelImage = localizeMessage('file_attachment.thumbnail', 'file thumbnail');
         if (fileInfo) {
             ariaLabelImage += ` ${fileInfo.name}`.toLowerCase();
         }
 
-        if (dimensions && dimensions.width && !this.state.loaded) {
+        const image = (
+            <img
+                {...props}
+                aria-label={ariaLabelImage}
+                tabIndex='0'
+                onKeyDown={this.onEnterKeyDown}
+                className={this.props.className}
+                src={src}
+                onError={this.handleError}
+                onLoad={this.handleLoad}
+                style={this.props.handleSmallImageContainer && this.state.isSmallImage ? {
```

Comment:
Can this be done with class names instead of inline styles?

Reply:
I would have loved to do it using a class, and actually tried multiple approaches before falling back to using in-line styles. There are multiple levels of inheritance and a specificity war is going on 🙈 🙃, and in such cases, I believe using in-line style is better than using `!important`.

It would be awesome and much appreciated If you could help with resolving the specificity issue without using `!important`.

Reply:
@abdusabri Thanks, but I'm not seeing the changes here

Reply:
The change is done locally but not pushed yet. Sorry if my comment was confusing.

Reply:
Okay no problem! :)